### PR TITLE
Reflect the cppcheck suggestions

### PR DESCRIPTION
**Pull request by BlaCkinkGJ**
The most severe problem was that the benchmark's `device_path` memory was not correctly set.

Note that some cppcheck suggestions are false alerts due to its bug([reference link](https://trac.cppcheck.net/ticket/10854))

```bash
cppcheck --quiet --enable=all --inconclusive -I include/ device/ramdisk/*.c   device/*.c util/*.c ftl/page/*.c interface/*.c *.c
device/ramdisk/ramdisk.c:324:2: error: Memory leak: ramdisk [memleak]
 return ret;
 ^
device/ramdisk/ramdisk.c:245:20: style: Variable 'addr.format.block' is assigned a value that is never used. [unreadVariable]
 addr.format.block = segnum;
                   ^
ftl/page/page-write.c:38:12: style: Variable 'paddr.lpn' is assigned a value that is never used. [unreadVariable]
 paddr.lpn = pgftl->trans_map[lpn];
           ^
nofile:0:0: information: Cppcheck cannot find all the include files (use --check-config for details) [missingInclude]
```

### CODE REVIEW

File: benchmark.c [hunk 1]
```diff
@@ -349,11 +349,10 @@ static struct benchmark_parameter *init_parameters(int argc, char **argv)
 
 	parm = (struct benchmark_parameter *)malloc(
 		sizeof(struct benchmark_parameter));
-	g_assert(parm != NULL);
 	memset(parm, 0, sizeof(struct benchmark_parameter));
 
 	device_path = parm->device_path;
-	memset(device_path, 0, sizeof(device_path) - 1);
```
[the remark this answers is not in the document]
Comment (everimind4) [reply benchmark.c:356]: device_path의 device는 Flashboard인가요?

**Review by everimind4 — APPROVED** [review]
기준씨, 수호씨가 할 때는 잘못된 Device Path로 인한 Memory 관련 Issue가 없었는지 궁금합니다.

File: benchmark.c (hunk 1 above)
[the remark this answers is not in the document]
Comment (BlaCkinkGJ) [reply benchmark.c:356]: This is only for Zoned Namespace SSD, not used in the Bluedbm. As I remembered, the Bluedbm device receives this value as NULL.

**Comment by BlaCkinkGJ** [comment]
> I wonder if there were any memory-related issues due to the wrong device path when Gijun and Suho were doing it.

If you pass the dynamically allocated string to this parameter, you must deallocate that string after using the Flash Translation Layer. In other cases, it does not have any issues. I suggest seeing each structure.

**Pull request merged**